email-service/src/main/resources/email.properties

```diff
@@ -0,0 +1,9 @@
+spring.mail.host=smtp.zoho.eu
```

Comment:
I don't think you meant to commit this file under this PR
This was the ignored file if I remember correctly